service/src/main/kotlin/com/integreety/activityengine/config/objectmapper/ObjectMapperCreator.kt

```diff
@@ -0,0 +1,18 @@
+package com.integreety.activityengine.config.objectmapper
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+
+class ObjectMapperCreator {
+    fun create():ObjectMapper {
+        val mapper = ObjectMapper()
+        mapper.registerModule(KotlinModule())
+        mapper.registerModule(JavaTimeModule())
+        mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+        return mapper
```

Comment:
You may have tried already this but wondering if worth enabling pretty-print here would help for the diagrams until we get that automated?